common/services/app/civic-uk/scripts.tsx

```diff
@@ -1,3 +1,38 @@
+import { font } from '@weco/common/utils/classnames';
+
+const branding = {
+  removeIcon: true,
+  removeAbout: true,
+  fontFamily: 'Inter, sans-serif',
+};
+
+const text = {
+  title: 'Manage cookies',
+  intro: '',
+  necessaryTitle: 'Essential cookies',
+  necessaryDescription:
+    'These cookies are necessary for our website to function and therefore always need to be on.',
+  notifyTitle: 'Our website uses cookies',
+  notifyDescription:
+    "We use cookies to make our website work. To help us make our marketing and website better, we'd like your consent to use cookies on behalf of third parties too.",
+  thirdPartyTitle: 'HI',
+  closeLabel: 'Close preference centre',
```

Comment:
This we need to discuss because the dismiss button isn't really in our plans but can't be removed from the actual preference centre, so we need to decide on copy. I made it a button so we have more control, but need to decide on that.

Reply:
I thought `notifyDismissButton` might do this, but it relies on being in the `notify` initial state?

![Screenshot 2024-03-01 at 10 43 40](https://github.com/wellcomecollection/wellcomecollection.org/assets/953792/460773ae-e568-4ebf-b97c-c9a844171eb3)

Reply:
That's just for the banner, which I set to false as I think we want to force a click on a button as much as possible. The `notify` prefix is used to configurate that first state (like `notifyTitle` or `notifyDescription`)